__tests__/app.test.js

```diff
@@ -34,10 +34,38 @@ describe('/GET', () => {
             expect(typeof response.body).toBe('object');
             expect(Object.keys(response.body).length).toBe(Object.keys(endpoints).length);
 
-            // This means the endpoints will always require a description!
             for(const key in response.body) {
-                expect(typeof response.body[key].description).toBe('string');
+                expect(response.body[key]).toEqual(endpoints[key]);
             }
         })
     })
+    describe('Articles', () => {
+        test('GET: 200 Return an object of the article with the matching article id', async () => {
+            const response = await request(app).get('/api/articles/1');
+            console.log(response.body);
+            expect(response.status).toBe(200);
+            expect(response.body).toMatchObject({
```

Comment:
We can and should be more specific when asserting the response object here, we know what the values should look like and so we should be more specific.